src/pg/brc20/brc20-pg-store.ts

```diff
@@ -264,53 +253,49 @@ export class Brc20PgStore extends BasePgStoreModule {
             OR (l.block_height = ${location.block_height} AND l.tx_index < ${location.tx_index})
```

Comment:
Unclear about this (old) line. _(the second OR condition)_
The transfer is not valid if there is a location (transfer/inscribe) in the same block, but that came before the origin tx?

Reply:
This is just looking for transfers of this inscription that happened before the current point in time (either at a past block OR at this same block but in an earlier transaction)